### DISCUSSION TITLE
quarto not working with julia 1.11.0

### DISCUSSION THREAD

**Discussion by kiritved** [quarto-dev/quarto-cli#11046] [Q&A] [answered]
### Description

i have a quarto project which uses r,julia & python chunks.
it was running sucessfully till i lupdated my julia from version 1.10.5 to 1.11.0
now it shows an error while executing julia chunk,but sucessfully runs R & python chunk
error displayed is given below
processing file: jr.qmd
  |...............                                                |  24% [j0]  Julia version 1.11.0 at location C:\Users\DELL\.julia\juliaup\julia-1.11.0+0.x64.w64.mingw32\bin will be used.
Error:
! jl_stdout_obj - The specified procedure could not be found.
Backtrace:
  1. global .main()
  2. execute(...)
  3. rmarkdown::render(...)
  4. knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet)
       at rmarkdown/R/render.R:768:5
     ...
 10. knitr:::block_exec(params)
 13. knitr (local) engine(options)
 14. JuliaCall::eng_juliacall(options)
 15. JuliaCall::julia_setup()
 16. JuliaCall:::juliacall_initialize(...)

Quitting from lines 151-156 [j0] (jr.qmd)
                                                                                                 
Execution halted

any one can help me on this?

**Comment by mcanouil** [comment]
Since you are using `knitr`, it means you need JuliaCall which might not yet support the last version of Julia
- https://github.com/Non-Contradiction/JuliaCall/issues

Regarding the Julia engine:

- https://github.com/quarto-dev/quarto-cli/issues/11013

In the future, please read and follow the guidelines given when opening new issues/discussions such as provided a small reproducible example formatting your message by putting code inside code blocks.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by cderv** [reply, marked as answer]
I believe the issue related to JuliaCall and Julia 1.11 is there:
- https://github.com/Non-Contradiction/JuliaCall/issues/234

**Comment by kiritved** [comment]
by reinstalling Julia 1.10.5,, the issue is resolved temporarily
REGARDS,


*KIRIT B. VEDTECHNICAL, MANAGEMENT & COMPUTER
CONSULTANT9167551554/9869015508*
*https://kiritved.com <http://kiritved.wixsite.com/kbv3012>*
***@***.***
***@***.***

**Comment by kiritved** [comment]
Thank you for your help!
REGARDS,


*KIRIT B. VEDTECHNICAL, MANAGEMENT & COMPUTER
CONSULTANT9167551554/9869015508*
*https://kiritved.com <http://kiritved.wixsite.com/kbv3012>*
***@***.***
***@***.***
